app/src/main/res/layout/layout_weekday_rv.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
since this is an item of the RecyclerView, rename the file as `lecture_item.xml`